app/webpacker/packs/active_admin.js

```diff
@@ -1,7 +1,13 @@
-// Load Active Admin's styles into Webpacker,
-// see `active_admin.scss` for customization.
+// Load ActiveAdmin’s styles into Webpacker.
+// See `active_admin.scss` for customisation.
 import "../styles/active_admin";
 
-import "@activeadmin/activeadmin";
+import Rails from '@rails/ujs'
```

Comment:
Do we need this? We're not actually using `ujs` or `activestorage` so unless this introduces them can probably remove them here.